### PR TITLE
feat(feedback): set feedback issue to ignored if spam

### PR DESCRIPTION
**Pull request by JoshFerge**
- If the message is spam, set it to be ignored by producing a status change message to the occurrence consumer.

There is not a way currently to set an issues state to ignored upon creation, but this should set it to ignored pretty close to immediate.

Note that there is sometimes buggy with issue occurrence messages sometimes processed out of order, but there is some ongoing work with issue platform to fix this soon.

### CODE REVIEW

File: src/sentry/feedback/usecases/create_feedback.py [hunk 1]
```diff
@@ -240,6 +241,16 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     produce_occurrence_to_kafka(
         payload_type=PayloadType.OCCURRENCE, occurrence=occurrence, event_data=event_fixed
     )
+    if is_message_spam:
+        produce_occurrence_to_kafka(
```
Comment (snigdhas) [inline src/sentry/feedback/usecases/create_feedback.py:245]: Not sure if this is the bug you're referring to but is it possible for this message to be processed before the `PayloadType.OCCURRENCE` message? would the group not exist in that case?

Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:245]: yes, that's the bug i'm referring to, and that's what steve is fixing here: https://github.com/getsentry/sentry/pull/69466

**Review by wedamija — COMMENTED** [review]
If we set the status to ignored after we create it, are we going to send notifications and so on?

**Comment by JoshFerge** [comment]
> If we set the status to ignored after we create it, are we going to send notifications and so on?

https://github.com/getsentry/sentry/pull/69478

**Review by wedamija — APPROVED** [review]

**Pull request merged**